src/main/java/tutoraid/commons/core/Messages.java

```diff
@@ -32,4 +27,13 @@
             + " entries to delete from.";
     public static final String MESSAGE_CAPACITY_LESS_THAN_STUDENTS = "The given capacity is lower "
             + "than the number of students currently enrolled in the lesson!";
+    public static final String MESSAGE_JSON_INTEGRITY_LESSON_OVER_CAPACITY = "The save data is corrupted: One or more "
```

Comment:
This String does not seem to be used anywhere, so maybe this unused variable can be removed?